lib/raven/configuration.rb

```diff
@@ -311,7 +311,10 @@ def capture_in_current_environment?
     end
 
     def server_configured?
-      host && path
+      verify!
```

Comment:
Will move the settings to a constant.

With `verify!` returning true/false, we would loose the descriptive errors (e.g. "No server specified"). We could return an array of messages from `verify!` and then raise the first one if not empty (although I'm not sure if I really like that - I guess no ;) ).